### DISCUSSION TITLE
MULLER led lamp Tasmota setup...newbye

### DISCUSSION THREAD

**Discussion by vinc59** [arendst/Tasmota#10489] [Q&A] [answered]
Hey all, I started playing with IoT and ended up to Tasmota r ecently. I found an excellent tool easy to manage. I am using Tasmota on ESO8266 devices and it works like a charm. I am a newbye so I miss most of informations. I am trying to integrate a Muller led lamp inTasmota tohave ti controlled by Alexa. However I do not understand how to set up and make the connection.
What settings do I have to set in Alexa if using a nodeMCU?
It seems iit is necessary to configure as Hue Bridge..I did it, but I miss the next step. Connection between lamp and system.
Thanks for You help.

**Comment by ascillato** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
